src/shared/openssl-util.c

```diff
@@ -3,6 +3,7 @@
 #include "openssl-util.h"
 #include "alloc-util.h"
 #include "hexdecoct.h"
+#include "random-util.h"
```

Comment:
hmm, not the fault of this PR, but we should order include lists alphabetically, can you fix this, if you touch this list?